skills_ml/utils/__init__.py

```diff
@@ -1 +1,17 @@
 """Common utilities"""
+def safe_get(dct, *keys):
```

Comment:
Although it's a bit more work, while we have the `skills_utils` repository we should probably put super generic utilities like this there. So it doesn't slow things down too much, you can probably push them straight to skills_utils master.